packages/mux-player/src/template.ts

```diff
@@ -108,66 +91,58 @@ export const ChromeRenderer = (props: MuxTemplateProps) => {
   return VodChromeSmall(props);
 };
 
-const MediaPlayButton = () => `
+const MediaPlayButton = () => html`
   <media-play-button>
     ${icons.Play({ slot: "play", title: "Play" })}
     ${icons.Pause({ slot: "pause", title: "Pause" })}
   </media-play-button>
 `;
 
-const MediaSeekBackwardButton = (props: Partial<MuxTemplateProps>) => `
+const MediaSeekBackwardButton = (props: Partial<MuxTemplateProps>) => html`
   <media-seek-backward-button seek-offset="${props.backwardSeekOffset}">
     ${icons.SeekBackward({ slot: "backward", value: props.backwardSeekOffset })}
   </media-seek-backward-button>
 `;
 
-const MediaSeekForwardButton = (props: Partial<MuxTemplateProps>) => `
+const MediaSeekForwardButton = (props: Partial<MuxTemplateProps>) => html`
   <media-seek-forward-button seek-offset="${props.forwardSeekOffset}">
     ${icons.SeekForward({ slot: "forward", value: props.forwardSeekOffset })}
   </media-seek-forward-button>
 `;
 
-const MediaMuteButton = () => `
+const MediaMuteButton = () => html`
   <media-mute-button>
     ${icons.VolumeHigh({ slot: "high" })} ${icons.VolumeLow({ slot: "medium" })}
     ${icons.VolumeLow({ slot: "low" })} ${icons.VolumeOff({ slot: "off" })}
   </media-mute-button>
 `;
 
-const MediaCaptionsButton = (props: MuxTemplateProps) => html`
-<media-captions-button${props.defaultShowCaptions ? " default-showing" : ""}>
+const MediaCaptionsButton = (
+  props: MuxTemplateProps
+) => html` <media-captions-button
+  default-showing="${props.defaultShowCaptions}"
+>
   ${icons.CaptionsOff({ slot: "off" })} ${icons.CaptionsOn({ slot: "on" })}
 </media-captions-button>`;
 
 const MediaAirplayButton = () => html`<media-airplay-button>
   ${icons.Airplay({ slot: "airplay" })}
 </media-airplay-button>`;
 
-const MediaPipButton = () => `<media-pip-button>
+const MediaPipButton = () => html`<media-pip-button>
   ${icons.Pip({ slot: "enter", title: "Enter Picture-in-Picture" })}
   ${icons.Pip({ slot: "exit", title: "Exit Picture-in-Picture" })}
 </media-pip-button>`;
 
-const MediaFullscreenButton = () => `<media-fullscreen-button>
+const MediaFullscreenButton = () => html`<media-fullscreen-button>
   ${icons.FullscreenEnter({ slot: "enter", title: "Enter Fullscreen" })}
   ${icons.FullscreenExit({ slot: "exit", title: "Exit Fullscreen" })}
 </media-fullscreen-button>`;
 
 export const VodChromeSmall = (props: MuxTemplateProps) => html`
   <media-control-bar slot="top-chrome" style="justify-content: flex-end;">
-    ${renderable(
-      "captionsButton",
-      ({ hasCaptions }: Partial<MuxTemplateProps>) =>
-        hasCaptions && MediaCaptionsButton(props),
-      props
-    )}
-    ${renderable(
-      "airplayButton",
-      ({ supportsAirPlay }: Partial<MuxTemplateProps>) =>
-        supportsAirPlay && MediaAirplayButton(),
-      props
-    )}
-    ${MediaPipButton()}
+    ${props.hasCaptions ? MediaCaptionsButton(props) : ""}
```

Comment:
non-blocking: This also feels a little clunky, but I think I'm more ok with it than other callouts ("render a Node/set of nodes if some condition is met, otherwise render an empty string" seems reasonable)

Reply:
we could also make it that all `false` values convert to an empty string in a node part. 
the case we need to print `false` is probably very low and then casting it to a string will work.